README.md

```diff
@@ -6,15 +6,32 @@ SourceTracker was originally described in [Knights et al., 2011](http://www.ncbi
 If you use this package, please cite the original SourceTracker paper linked
 above pending publication of SourceTracker 2.
 
+# File Formats
+
+For descriptions of all file formats and options, please see the help
+documentation, available with the command `sourcetracker2 gibbs --help`. 
+
+In brief, this script requires a feature X sample contingency table
+(traditionally an OTU table) and sample metadata (traditionally a mapping file).
+Any feature table that can be read by the `biom` package will be acceptable
+input. For example file formats please look at the test mapping file and
+feature (OTU) tables we have included at `sourcetracker2/data/tiny-test/`.
+
+# Preprocessing
+
+Input feature data should be counts. If non-count data (e.g. the count of
+feature i in sample j was 4.63) is passed, the 'ceiling' of the data will be
+taken. This means that each non-integer count will be rounded up to the nearest
+larger integer.
```

Comment:
I don't know that the statement by @lkursell is necessary, but rather that it's implied here (since it says that only counts are accepted). Another way to go would be to fail if something other than integer counts are provided. I think that might be preferable, since they're really doing something wrong if a floating point value is encountered. numpy should fail if you try to cast an array to int when there are values that cannot be safely converted to ints.